common/trainer/default_trainer.py

```diff
@@ -57,8 +57,10 @@ def recovery(self, config):
                 self.starting_epoch = checkpoint_epoch["epoch"] + 1
 
             logger.info(f"recovery to epoch {self.starting_epoch}")
+        except FileNotFoundError as e:
+            logger.info(e)
```

Comment:
is it correct?

Reply:
Currently will print a ValueError if no checkpoint file found, this update will change ValueError to a normal info log